hyperliquid/exchange_plus.py

```diff
@@ -0,0 +1,106 @@
+from eth_account.signers.local import LocalAccount
+
+from hyperliquid.info import Info
+from hyperliquid.exchange import Exchange
+from hyperliquid.utils.types import Any, List, Literal, Meta, Optional, Tuple, Cloid
+
+# Extends Exchange Class with 'Market Orders'
+class ExchangePlus(Exchange):
+    DEFAULT_SLIPPAGE = 0.05 # Max allowed Slippage for Market Orders
+
+    def __init__(
+        self,
+        wallet: LocalAccount,
+        base_url: Optional[str] = None,
+        meta: Optional[Meta] = None,
+        vault_address: Optional[str] = None,
+    ):
+        super().__init__(wallet, base_url, meta, vault_address) 
+        self.info = Info(base_url, skip_ws=True)
+
+        # create a szDecimals map
+        self.sz_decimals = {}
+        for asset_info in self.meta["universe"]:
+            self.sz_decimals[asset_info["name"]] = asset_info["szDecimals"]        
+
+    def __round_sz(
+        self,
+        coin: str,
+        sz: float                
+    ) -> float:
+        # we round sz based on the sz_decimals map
+        return round(sz, self.sz_decimals[coin])
+
+    def __round_px(
```

Comment:
I think you might as well just include this in slippage price since it's only called there and it's one line

Reply:
yep